src/color_picker_cell.coffee

```diff
@@ -0,0 +1,25 @@
+###*
+ * ColorPickerCell
+ *
+ * This is an internal-use view to control the logic of a color cell in
+ * the color picker.
+ * @class
+ * @augments {Ember.View, Ember.Widgets.StyleBindingMixin,
+ *            Ember.Widgets.ColorPickerMixin}
+###
+Ember.Widgets.ColorPickerCellComponent =
```

Comment:
I think we can pass in any attributes belonging to the component, right? I guess we should put comment on the attributes inside the component. This code was moved from existing code (splitting the file) that's why there is no comment here.